servers/src/main/java/tachyon/master/file/journal/InodeFileEntry.java

```diff
@@ -52,11 +78,22 @@ public InodeFileEntry(@JsonProperty("creationTimeMs") long creationTimeMs,
     mTTL = ttl;
   }
 
+  /**
+   * Converts the entry to {@link InodeFile}.
+   *
+   * @return the {@link InodeFile} representation
+   */
   public InodeFile toInodeFile() {
     InodeFile inode =
-        new InodeFile.Builder().setName(mName).setBlockContainerId(BlockId.getContainerId(mId))
-            .setParentId(mParentId).setBlockSizeBytes(mBlockSizeBytes)
-            .setCreationTimeMs(mCreationTimeMs).setTTL(mTTL).setPersisted(mPersisted).build();
+        new InodeFile.Builder()
+            .setName(mName)
+            .setBlockContainerId(BlockId.getContainerId(mId))
+            .setParentId(mParentId)
+            .setBlockSizeBytes(mBlockSizeBytes)
+            .setCreationTimeMs(mCreationTimeMs)
+            .setTTL(mTTL)
```

Comment:
oh, I just found it inconsistent: mPersisted, mPinned etc do not have setter in the builder